unidist/core/backends/mpi/core/communication.py

```diff
@@ -513,32 +536,37 @@ def recv_complex_data(comm, source_rank):
 # ---------- #
 
 
-def send_complex_operation(comm, operation_type, operation_data, dest_rank):
+def send_simple_operation(comm, operation_type, operation_data, dest_rank):
     """
-    Send operation and data that consist of different user provided complex types, lambdas and buffers.
-
-    The data is serialized with ``unidist.core.backends.mpi.core.ComplexDataSerializer``.
+    Blocking send an operation and standard Python data types.
 
     Parameters
     ----------
     comm : object
         MPI communicator object.
-    operation_type : ``unidist.core.backends.mpi.core.common.Operation``
+    operation_type : unidist.core.backends.mpi.core.common.Operation
         Operation message type.
     operation_data : object
         Data object to send.
     dest_rank : int
         Target MPI process to transfer data.
+
+    Notes
+    -----
+    Serialization is a simple pickle.dump in this case.
+    A blocking send is used when waiting for a completion that is necessary
+    for the pipeline to continue, or when the receiver is waiting for a result.
+    Otherwise use non-blocking function ``isend_simple_operation``
```

Comment:
```suggestion
    * This blocking send is used when we have to wait for completion of the communication,
    which is necessary for the pipeline to continue, or when the receiver is waiting for a result.
    Otherwise, use non-blocking ``isend_simple_operation``.
    * Serialization of the data to be sent takes place just using ``pickle.dump`` in this case.
```